job.go

```diff
@@ -255,6 +256,25 @@ func (j *AuroraJob) AddURIs(extract bool, cache bool, values ...string) Job {
 	return j
 }
 
+// AddURI adds a single URI to to fetch while the task is bootstrapping. Unlike AddURIs, this function allows
```

Comment:
You may want to remove the duplicated preposition

Reply:
Good catch, fixing it